var/spack/repos/builtin/packages/adios2/package.py

```diff
@@ -76,6 +76,11 @@ class Adios2(CMakePackage, CudaPackage):
     variant("dataspaces", default=False, when="@2.5:", description="Enable support for DATASPACES")
     variant("ssc", default=True, description="Enable the SSC staging engine")
     variant("hdf5", default=False, description="Enable the HDF5 engine")
+    variant(
+        "aws",
+        default=False,
+        description="Enable support for S3 compatible storage using AWS SDK's S3 module",
+    )
```

Comment:
Given the PR description it sounds like you probably want to add `when="@2.9:",` argument to the variant.